static/scss/homepage.scss

```diff
@@ -199,68 +209,72 @@ body.app-media {
   }
 }
 
-.sub-banner-section{
-  background-color: #ffffff;
-  padding: 7vw 0 8vw;
-  text-align: center;
-}
-.sub-banner-text {
-  text-align:left;
-  margin: 0 auto;
-  color:#000;
-  font-size:25px;
-  font-weight:200;
-  line-height: 44px;
-  width: 81%;
-  max-width:1000px;
-  @media (max-width: 580px) {
-    width: 90%;
-  }
+.sub-banner {
+  background-color: #e6e6e6;
 
-  h3 {
-    font-size: 25px;
-    @media (min-width: 1200px) {
-    font-size: 1.9vw;
-    }
-    margin: 0 0 30px;
-    font-weight: 400;
-    color: rgba(0,0,0,.87);
-  }
-  p {
-    font-size: 16px;
-    @media (min-width: 1200px) {
-    font-size: 1.2vw;
-    }
-    font-weight: 400;
-    color: rgba(0,0,0,.6);
-  }
-}
+  .sub-banner-section{
+    padding: 7vw 0 8vw;
+    text-align: center;
 
-.sub-banner-logos{
-  margin-top: 26px;
+    .sub-banner-text {
+      text-align:left;
+      margin: 0 auto;
+      color:#000;
+      font-size:25px;
+      font-weight:200;
+      line-height: 44px;
+      width: 81%;
+      max-width:1000px;
+      @media (max-width: 580px) {
+        width: 90%;
+      }
 
-  img{
-    margin-left: 5px;
-    width: 78px;
-    position: relative;
-    top: -8px;
-  }
-}
+      h3 {
+        font-size: 25px;
+        @media (min-width: 1200px) {
+        font-size: 1.9vw;
+        }
+        margin: 0 0 30px;
+        font-weight: 400;
+        color: rgba(0,0,0,.87);
+      }
 
-.sub-banner-logos-text {
-  vertical-align: bottom;
-  font-size: 16px;
-  display: inline-block;
-  font-size: 16px;
-  font-weight: 400;
-  color: rgba(0, 0, 0, 0.6);
-  @media (min-width: 1200px) {
-    font-size: 1.2vw;
-  }
-  @media (max-width: 580px) {
-    display: block;
-    line-height: 1.4em;
-    margin-bottom: 20px;
+      p {
+        font-size: 16px;
+        @media (min-width: 1200px) {
+        font-size: 1.2vw;
+        }
+        font-weight: 400;
+        color: rgba(0,0,0,.6);
+      }
+
+      .sub-banner-logos{
+        margin-top: 26px;
+        img{
+        margin-left: 5px;
+        width: 78px;
+        position: relative;
+        top: -8px;
+        }
+
+        .sub-banner-logos-text {
+          vertical-align: bottom;
+          font-size: 16px;
+          display: inline-block;
+          font-size: 16px;
+          font-weight: 400;
+          color: rgba(0, 0, 0, 0.6);
+          @media (min-width: 1200px) {
+            font-size: 1.2vw;
+          }
+          @media (max-width: 580px) {
+            display: block;
+            line-height: 1.4em;
+            margin-bottom: 20px;
+          }
+        }
+      }
+    }
```

Comment:
can you go through this section and add blank lines around all the blocks? for instance, there are a lot of media queries that look something like this:

```scss
rule: value;
other-rule: value;
@media (mid-width: foobarpx) {
  third-rule: value;
}
more-rule: value;
```

that should look more like:


```scss
rule: value;
other-rule: value;

@media (mid-width: foobarpx) {
  third-rule: value;
}

more-rule: value;
```